app/controllers/bz_controller.rb

```diff
@@ -0,0 +1,16 @@
+# This holds BZ custom endpoints for updating our
+# custom data.
+
+class BzController < ApplicationController
+
+  before_filter :require_user, :only => [:last_user_url]
+  skip_before_filter :verify_authenticity_token, :only => [:last_user_url]
+
+  def last_user_url
+    @current_user.last_url = params[:last_url]
```

Comment:
On Thu, Jul 02, 2015 at 08:46:35AM -0700, Brian Sadler wrote:
> Is it possible that there is no ```@current_user```?  I ran into that with the google analytics code and it's filling up the error logs.

The `before_filter :require_user` handles it here.